@here/harp-mapview/lib/MapView.ts

```diff
@@ -2956,6 +2964,40 @@ export class MapView extends THREE.EventDispatcher {
         const worldOffsetX = tile.computeWorldOffsetX();
         if (tile.willRender(zoomLevel)) {
             for (const object of tile.objects) {
+                if (object.userData.technique && object.userData.feature) {
+                    const technique: Technique = object.userData.technique;
+                    const objInfos = object.userData.feature.objInfos as ValueMap[];
+                    if (technique.hasOwnProperty("filter") && Array.isArray(objInfos)) {
+                        // ### TODO: evaluate only when properties references by the filter change.
+                        const geometry: THREE.BufferGeometry = (object as any).geometry;
+                        if (geometry && geometry.isBufferGeometry) {
+                            geometry.clearGroups();
+                            objInfos.forEach((properties, index) => {
```

Comment:
the representation of feature attributes (`objInfos`) needs to be way more efficient than what we have today.